updatecli/updatecli.d/helm-controller.yaml

```diff
@@ -0,0 +1,68 @@
+---
+name: "Bump Helm Controller version"
```

Comment:
This seems to be updating a go module, isn't that considered out of scope for this tool?

Reply:
I don't believe it's out of scope. Why do you think it's?

The idea is to target this kind of PR https://github.com/k3s-io/k3s/pull/6549/files.

Reply:
Makes sense. I'll update the README.